two_muscle_case/settings_contraction.py

```diff
@@ -25,7 +25,7 @@
 else:
   individuality_parameter = str(time.time())
 
-tendon_length_0 = variables.physical_offset
+tendon_length_0 = variables.physical_offset[2] + variables.zmin - variables.zmax
```

Comment:
Why not just `tendon_length_0 = variables.physical_offset[2] - variables.zmax`? 

Afaik physical_offset tells you the start of muscle 2, while zmax tells you the end of muscle 1

Reply:
sorry, i just understood this. start of muscle 2 = offset + zmin, hence this is correct